plugins/network/upnpc_

```diff
@@ -144,6 +147,14 @@ down.graph no
 up.negative down
 EOF
 			;;
+		"multi"|"upnpc")
```

Comment:
I think, it would be better to normalize the "MODE" value at the beginning (i.e. `[ "$MODE" = "upnpc" ] && MODE="multi"`). Otherwise the `"multi"|"upnpc"` conditional can be easily misunderstood.